CentralityiOS/CentralityiOS/ViewControllers/CentralityHelpers.m

```diff
@@ -58,10 +67,23 @@ + (void) updateLabel:(UILabel*)label newText:(NSString*)newText isHidden:(BOOL)i
 }
 
 + (PFQuery*)queryForUsersCompletedTasks{
-    PFQuery *allUserTasks = [PFQuery queryWithClassName:kTaskClassName];
-    [allUserTasks whereKey:kByOwnerQueryKey equalTo:PFUser.currentUser];
-    [allUserTasks whereKeyExists:kByDateCompletedKey];
-    return allUserTasks;
+    PFQuery *completedTasks = [PFQuery queryWithClassName:kTaskClassName];
+    [completedTasks whereKey:kByOwnerQueryKey equalTo:PFUser.currentUser];
+    [completedTasks whereKeyExists:kByDateCompletedKey];
+    [completedTasks whereKey:kIsCompletedKey equalTo:@(YES)];
+    return completedTasks;
+}
+
+// Queries tasks due within 24 hour margin of error.
```

Comment:
nice commenting! succinct and describes why you're doing it rather than just rewording what the code says.